rpcs3/Emu/Cell/lv2/sys_time.cpp

```diff
@@ -179,10 +183,46 @@ u64 get_guest_system_time()
 // Functions
 error_code sys_time_get_timezone(vm::ptr<s32> timezone, vm::ptr<s32> summertime)
 {
-	sys_time.warning("sys_time_get_timezone(timezone=*0x%x, summertime=*0x%x)", timezone, summertime);
+	sys_time.notice("sys_time_get_timezone(timezone=*0x%x, summertime=*0x%x)", timezone, summertime);
 
-	*timezone   = 180;
-	*summertime = 0;
+#ifdef _WIN32
+	TIME_ZONE_INFORMATION tz{};
+	switch (GetTimeZoneInformation(&tz))
+	{
+	case TIME_ZONE_ID_UNKNOWN:
+	{
+		*timezone = -tz.Bias;
+		*summertime = 0;
+		break;
+	}
+	case TIME_ZONE_ID_STANDARD:
+	{
+		*timezone = -tz.Bias;
+		*summertime = -tz.StandardBias;
```

Comment:
I don't really understand the standard bias. But I'd assume that it should be added to the bias and not counted as summertime.